scripts/exglobal_diag.sh

```diff
@@ -232,9 +232,12 @@ EOFdiag
    # Restrict diagnostic files containing rstprod data
    rlist="conv_gps conv_ps conv_pw conv_q conv_sst conv_t conv_uv saphir"
    for rtype in $rlist; do
-      set +e
-      ${CHGRP_CMD} *${rtype}*
-      ${STRICT_ON:-set -e}
+      for rfile in *"${rtype}"*; do 
+         if [[ -s "${rfile}" ]]; then
+            ${CHGRP_CMD} "${rfile}"
+         fi
+	 ${STRICT_ON:-set -e}
```

Comment:
Why is this needed?

Reply:
If `set +e` is removed, it's not. There is no need to turn back on something you never turned off.

Reply:
@WalterKolczynski-NOAA noted that `set +e` was no longer needed. Is this? I would assume not but @aerorahul noted above the following https://github.com/NOAA-EMC/global-workflow/pull/2688#discussion_r1640603759.

Reply:
Fixed.